.github/workflows/quality.yml

```diff
@@ -23,12 +24,56 @@ jobs:
         run: |
           ./gradlew ktlintCheck --continue
       - name: Upload reports
+        if: always()
         uses: actions/upload-artifact@v2
         with:
           name: ktlinting-report
-          path: vector/build/reports/ktlint/*.*
+          path: |
+            */build/reports/ktlint/ktlint*/ktlint*.txt
+      - name: Handle Results
+        if: always()
+        id: get-comment-body
+        run: |
+          results="$(cat */*/build/reports/ktlint/ktlint*/ktlint*.txt */build/reports/ktlint/ktlint*/ktlint*.txt | sed -r "s/\x1B\[([0-9]{1,3}(;[0-9]{1,2})?)?[mGK]//g")"
+          if [ -z  "$results" ]; then
+            body="👍 ✅ 👍"
+          else
+            body="\`👎 ❌ 👎 ${results}\`"
```

Comment:
Small remark, the Emojis could be outside of the "`", no?

Reply:
Sure it would, however if you observe commenting with that lib do not support `\n` so I had to manually create the new lines by replacing via bash the aggregated results with `<br>`. But then I realised that **code blocks** do not support <br> new lines! 😵‍💫 So to support **newlines** along with code blocks I added multiple `code blocks` around `<br>`. So there should be an init `left_code_block` in order to work.

It was the tricky to support newlines with **codeblocks** without the lib supporting it

The best I can do is (that seems pretty nice):
👎 ❌ 👎`Failed `

Reply:
Ah, thanks for the explaination.
Maybe do not add Emoji there but use the `reactions`, different depending on the result?
Anyway this is fine like this.